admin/worker/delete_unsued_tokens.go

```diff
@@ -0,0 +1,23 @@
+package worker
```

Comment:
Can you create the job in `admin/jobs/river` instead? We're migrating to that for background jobs and eventually will remove the `worker` package.

https://github.com/rilldata/rill/blob/d24cf0179d58b96af81ac256c282751bb925fe98/admin/jobs/river/river.go#L96

Reply:
Can do, want me to create an issue for that?